zebra-chain/src/transaction/txid.rs

```diff
@@ -43,10 +43,14 @@ impl<'a> TxIdBuilder<'a> {
         Ok(Hash(hash_writer.finish()))
     }
 
-    /// Compute the Transaction ID for a V5 transaction in the given network upgrade.
+    // FIXME: it looks like the updated zcash_primitives in librustzcash
+    // auto-detects the transaction version by the first byte, so the same function
+    // can be used here for both V5 and V6.
```

Comment:
remove these 3 lines

Reply:
`txid_v5_v6()`'s approach _would_ work for all transactions, but it's an unnecessary memory/cache allocation when `txid_v1_to_v4()` is so simple and already implemented.